### DISCUSSION TITLE
Implementing a simple following system, question about controllers and views

### DISCUSSION THREAD

**Discussion by prp-e** [jonhue/acts_as_favoritor#177] [Q&A] [answered]
You have documented the whole model and initializer thing very well, but I couldn't find proper documentations about controllers or views. 
It'd be awesome if you add them in the `README.md` file as well. 
Thanks!

**Comment by jonhue** [comment, marked as answer]
This gem is neither built specifically for a MVC type of app nor Ruby on Rails. It solely depends on ActiveRecord. It is supposed to be a backend to a favorite system. What frontend is used to interact with that backend and how this interaction works is left to the user of this library. If you do not know where to start, I would suggest looking into some basic Rails tutorials (I am assuming you are using Rails).
